cython_lint/cython_lint.py

```diff
@@ -854,14 +857,14 @@ def main(argv: Sequence[str] | None = None) -> int:  # pragma: no cover
     parser.add_argument('--version', action='version', version=__version__)
     parser.add_argument(
         '--ignore',
-        type=str,
+        nargs='*',
         default='',
         help='Comma-separated list of pycodestyle error codes to ignore',
     )
     args = parser.parse_args(argv)
     ret = 0
 
-    ignore = {s.strip() for s in args.ignore.split(',') if s.strip() != ''}
+    ignore = set(args.ignore)
```

Comment:
I tested and this works but in a surprising way. Indeed at this point, ignore becomes a set with only 1 string element, e.g. ``{"E721,E731"}``. It seems that pycodestyle post processes it appropriately anyway but it might be less confusing from a maintenance point of view to do ``set(s.strip() for s in args.ignore)``

Reply:
you're right, thanks, I should've just kept it as you'd written it, apologies (even though it does seem to work fine)